shell/components/templates/standalone.vue

```diff
@@ -1,7 +1,3 @@
-<script>
```

Comment:
This is for standalone plugins. They should have their own branding so I didn't switch this with the Brand mixin.